client/my-sites/stats/controller.jsx

```diff
@@ -369,6 +369,7 @@ export function site( context, next ) {
 			chartTab={ chartTab }
 			context={ context }
 			period={ rangeOfPeriod( activeFilter.period, date ) }
+			statsPurchaseSuccess={ queryOptions.statsPurchaseSuccess }
```

Comment:
What's the rationale for using `statsPurchaseSuccess` as opposed to `purchaseSuccess`?

Reply:
There isn't really a rationale here, other than clarity on the product being purchased. However, in this context, I suppose that's fairly given already. Do you think `purchaseSuccess` would be a better fit?

Reply:
Either one works! Feel free to go with whichever one you feel would be appropriate.